nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server.hpp

```diff
@@ -86,6 +86,14 @@ class BtActionServer
    */
```

Comment:
The builds are failing, something isn't linking correctly from these changes

Reply:
Oh, this PR doesn't have the `.cpp` implementation of `setGrootMonitoring` anymore, odd